src/FakeItEasy/ReturnValueConfigurationExtensions.cs

```diff
@@ -117,7 +117,7 @@ public static void ReturnsNextFromSequence<T>(this IReturnValueConfiguration<Tas
             configuration.ReturnsNextFromQueue(new ConcurrentQueue<Task<T>>(values.Select(TaskHelper.FromResult)));
         }
 
-        private static void ReturnsNextFromQueue<T>(this IReturnValueConfiguration<T> configuration, ConcurrentQueue<T> queue)
+        internal static void ReturnsNextFromQueue<T>(this IReturnValueConfiguration<T> configuration, ConcurrentQueue<T> queue)

```

Comment:
To make it accessible to FakeItEasy.Extensions.ValueTask. If you think it's not worth it, I can just duplicate the code and remove the `InternalsVisibleTo`